app/components/setup/boundaries/index.js

```diff
@@ -62,16 +92,31 @@ class SetupBoundaries extends Component {
             </TouchableHighlight>
           </View>
         </View>
-        <ProtectedAreas
-          iso={'VEN'}
-          visible={this.state.showProtectedAreas}
-        />
+        {this.state.showProtectedAreas &&
```

Comment:
As we have 3 components in the setup slider I've used the flag to hide the Areas map instead the modal visible prop to try to keep it as light as possible